src/lib/y2partitioner/actions/delete_bcache.rb

```diff
@@ -35,6 +35,28 @@ def initialize(*args)
 
     private
 
+      # @see DeleteDevice#errors
+      def errors
+        errors = super + [shared_cset]
+
+        errors.compact
+      end
+
+      # Error when the bcache shares cset with any other bcache.
+      #
+      # @see doc/bcache.md
+      # @return [String, nil] nil if there is no error
+      def shared_cset
+        return nil unless device.bcache_cset
+        return nil if single_bcache_cset? || !device.exists_in_probed?
```

Comment:
In general it's preferred to use `Y2Partitioner::DeviceGraph` to decouple the code from the existence of the "holy and omnipresent" devicegraphs `probed` and `staging`.

Like here https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2partitioner/actions/edit_md_devices.rb#L140